py/src/braintrust/framework.py

```diff
@@ -368,7 +368,7 @@ def meta(self, **info):
 
 
 def init_experiment(project_name, experiment_name: Optional[str] = None, metadata: Optional[Metadata] = None):
-    ret = _init_experiment(project_name, experiment=experiment_name, metadata=metadata)
+    ret = _init_experiment(project_name, experiment=experiment_name, metadata=metadata, set_current=False)
```

Comment:
do we need to do similar stuff in JS?

Reply:
As of yet the JS SDK still uses async local storage for experiments/loggers, so this isn't a problem.